app/Parser.tsx

```diff
@@ -376,8 +385,6 @@ function Details({
   );
 }
 
-function RawJson({ data }: { data: string }) {
-  const json = JSON.parse(data);
-
-  return <JSONTree data={json} shouldExpandNodeInitially={() => true} />;
+function RawJson({ data }: { data: any }) {
```

Comment:
not sure what to do about this one -- currently, this displays our IR, which probably isn't quite what we want

Reply:
This this is only used in TabPanelGenericData though, which is at the root of a tab, so having the raw json accessible from there should not be as hard as for each level of the tree.